systemd/systemd.go

```diff
@@ -34,6 +34,7 @@ import (
 
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/osutil"
+	"github.com/snapcore/snapd/snap/squashfs"
```

Comment:
this dependency feels a bit strange from generic systemd to snap/squashfs,  wondering if the fstype helpers belong somewhere else

Reply:
Yeah, I was thinking about this too, ideas very welcome. Maybe `osutil` ? But that feels a bit too generic.

Reply:
osutil/squashfs ?  too much too of small packages?

Reply:
maybe a simpler option is to leave that there, but change the interface here to:

WriteMountUnitFile(name, revision, what, where, fstype string, extraOptions []string)

and call squashfs.FsType from the caller, there's anyway only one caller

Reply:
I think I slightly prefer moving to `osutil/squashfs` to keep the knowledge about squashfs handling in a self-contained block. But if you have strong opinions to go with option (2) (changing WriteMountUnitFile) I'm fine with this as well.